src/styles/blocks/card.scss

```diff
@@ -0,0 +1,86 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+
+  &__image {
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+    display: block;
+  }
+
+  &__description {
+    color: $main-text-color;
+    margin-top: 40px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    width: 166px;
```

Comment:
The width of the description is set to 166px. It's better to let it take the remaining space of the parent container. You can achieve this by setting width to 100% or by not setting it at all, as block-level elements like divs take up the full width of their parent by default. This will make the layout more flexible and adaptable to changes in the parent container's size.